### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by jeremyvergnas**
doc(project): update docs

### CODE REVIEW

**Comment by RWOverdijk** [comment]
@ppvg satisfied? :) I am.

**Review by RWOverdijk — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -34,7 +34,9 @@ Here's a snippet to give you an idea of what this module supports.
 @bindable columns = [
   {name : "id",         value: "id",        type: 'number'},
   {name : "Name",       value: "name",      type: 'text'},
-  {name : "Created at", value: "createdAt", type: 'date'}
+  {name : "Created at", value: "createdAt", type: 'date'},
+  // For enumeration
+  {name : "State", value: "sate", type: 'select'}
```
Comment (ppvg) [inline README.md]:
```diff
-  {name : "State", value: "sate", type: 'select'}
+  {name : "State", value: "state", type: 'select'}
```

Same in `doc/usage.md`.

**Review by ppvg — COMMENTED** [review]
Where does the `options` property go? On the filter object in `columns`?

**Comment by ppvg** [comment]
@RWOverdijk please let me know if I'm being too critical in review. I'm willing to be more yielding if you want.

**Comment by jeremyvergnas** [comment]
@ppvg I don't understand your question "Where does the options property go? On the filter object in columns?"

`options` are generated based on the enum values that we get out of the entity's. Don't know if this answer your question.

**Comment by ppvg** [comment]
Ah, right. Got it. Is that immediately obvious to people that might use this library? (Honest question; 'cause I wouldn't know.) If not; please document _that_. Otherwise: yes, satisfied. 😛

**Comment by jeremyvergnas** [comment]
@ppvg documented

**Comment by ppvg** [comment]
Nice.

**Pull request merged**